### PR TITLE
Use data in followup questions

### PR DESCRIPTION
**Pull request by baruchoxman**

### CODE REVIEW

File: src/streamlit_app.py [hunk 1]
```diff
@@ -298,13 +301,34 @@ def push_assistant_query_result(
...
+        key = str(uuid.uuid4())[:8]
         msg = {
             "type": HistoryItemTypes.QUERY_RESULT.value,
             "role": Roles.ASSISTANT.value,
             "data": df,
+            "key": key,
             "text": "",
         }
 
+        if 0 < len(df) < _MAX_VALUES_TO_USE_FOR_FOLLOWUP_QUESTION:
```
Comment (krakov) [inline src/streamlit_app.py:313]: would you want to say "no data" to the llm when none is found? so it can make conclusions based on that

**Review by krakov — APPROVED** [review]

File: src/streamlit_app.py (hunk 1 above)
Comment (baruchoxman) [reply src/streamlit_app.py:313]: I can send an empty json, with the columns and everything (basically - remove the check condition of greater than 0) - wdyt?

Comment (krakov) [reply src/streamlit_app.py:313]: probably fine

Comment (baruchoxman) [reply src/streamlit_app.py:313]: Fixed

**Pull request merged**